ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/validation/GossipValidationHelper.java

```diff
@@ -70,6 +71,17 @@ public boolean isSignatureValidWithRespectToProposerIndex(
         .orElse(false);
   }
 
+  public boolean isSignatureValidWithRespectToBuilderIndex(
```

Comment:
it's 100% the same as `isSignatureValidWithRespectToProposerIndex`, which is confusing

Reply:
Yeah right, it's confusing how builder and validator are interchangeable when it comes to signature veification. Fixed that by wrapping the method for the sake of readability